service/src/main/java/gov/nasa/pds/api/registry/opensearch/OpenSearchRegistryConnectionImpl.java

```diff
@@ -95,6 +97,9 @@ public HttpAsyncClientBuilder customizeHttpClient(
 						        SSLContext sslContext = sslBld.build();
 	
 						        httpClientBuilder.setSSLContext(sslContext);
+						        if (!sslCertificateVerification) {
+						        	httpClientBuilder.setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE);
```

Comment:
You only disable host name verification. Don't we also want to trust self-signed certificates?

Reply:
@tdddblog I did just what I needed for the api to work with the opensearch within docker network. If the hostname trick works maybe we can just dismiss this pull request.

Reply:
@tdddblog I changed the name of the property to be more explicit on the fact that we only disable the hostname verification.